resources/dicts/patrols/new_cat_hostile.json

```diff
@@ -51,93 +96,161 @@
     "season": "Any",
     "tags": ["border", "new_cat", "new_cat_injury", "nc_blunt_force_injury", "reputation", "no_change_fail_rep"],
     "intro_text": "r_c finds a wounded cat near the Thunderpath.",
+    "decline_text": "They leave the wounded cat alone.",
     "success_text": [
         "The patrol approaches the cat and sees some signs of life. The patrol, after some hesitation, decides to bring the cat back to camp so that they can be treated by the medicine cat. The cat thanks them profusely and promises to repay them by joining the Clan and being the best warrior they can."
     ],
     "fail_text": [
-        "As r_c inspects the cat, they find that they are already hunting with their ancestors.",
+        "As r_c inspects the cat, {PRONOUN/r_c/subject} find that they are already hunting with their ancestors.",
         "The cats attempt to carry the cat back to camp, but the cat passes on the way. They stop to bury the cat before continuing home."
     ],
-    "decline_text": "They leave the wounded cat alone.",
     "antagonize_text": "Your patrol drives the cat off of the territory. It's not c_n's problem.",
     "antagonize_fail_text": "The patrol tries to drive the cat away, but they trip and stumble over a loose twig. They lay on the ground and don't get back up. The patrol takes a bit of time to bury them - just to keep the scavengers away.",
     "chance_of_success": 40,
     "exp": 10,
     "win_skills": ["smart", "very smart", "extremely smart"],
-    "min_cats": 1,
+    "min_cats": 2,
     "max_cats": 6
 
+},
+{
+    "patrol_id": "gen_bord_newcatthunderpathsolo2",
```

Comment:
in order to make sure this works for the patrol sprite, you should remove 'solo' from the ID and just increment the number